corehq/apps/saved_reports/models.py

```diff
@@ -792,3 +829,98 @@ def get_latest(cls):
             return cls.objects.order_by('-end_datetime')[0]
         except IndexError:
             return None
+
+class SQLReportConfig(SyncSQLToCouchMixin, models.Model):
+    domain = models.CharField(max_length=32, null=False, db_index=True)
+    report_type = models.CharField(max_length=32, null=False)
+    report_slug = models.CharField(max_length=32, null=False)
+    subreport_slug = models.CharField(max_length=64, null=True)
+    name = models.CharField(max_length=128, null=False)
+    description = models.CharField(max_length=1000, null=True)
+    owner_id = models.CharField(max_length=64, null=False)
+    filters = JSONField(null=False, default=dict)
+    date_range = models.CharField(max_length=12, null=True)
+    days = models.IntegerField(null=True)
+    start_date = models.DateField(null=True)
+    end_date = models.DateField(null=True)
+    datespan_slug = models.CharField(max_length=64, null=True)
+
+    # unknown fields
+    update_seq = models.CharField(max_length=1000, null=False)
+    purge_seq = models.IntegerField(null=False)
+    db_name = models.CharField(max_length=12, null=False)
+    doc_del_count = models.IntegerField(null=False)
+    instance_start_time = models.CharField(max_length=1, null=False)
+    disk_size = models.IntegerField(null=False)
+    doc_count = models.IntegerField(null=False)
+    disk_format_version = models.IntegerField(null=False)
+    data_size = models.IntegerField(null=False)
```

Comment:
are these necessary?

Reply:
I was not sure about what purpose do these fields would be serving. I found some references of the fields in some pillow files but not entirely sure if they belong to ReportConfig. So I kept them with a comment.

Reply:
I think these should be deleted if there are no meaningful code references to them.